packages/googlechrome.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,55 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\sfta.ps1
+. $toolsPath\helpers.ps1
+
+$downloadDir = Get-PackageCacheLocation
+$installer          = 'googlechromeinstaller.msi'
+$packageArgs        = @{
+    packageName     = $env:ChocolateyPackageName
+    file            = Join-Path $downloadDir $installer
+    url             = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise.msi'
+    url64bit        = 'https://dl.google.com/tag/s/dl/chrome/install/googlechromestandaloneenterprise64.msi'
+    fileType        = 'MSI'
+    silentArgs      = "/quiet /norestart /l*v `"$($env:TEMP)\$($env:chocolateyPackageName).$($env:chocolateyPackageVersion).MsiInstall.log`""
+    validExitCodes  = @(0)
+}
+
+# Download the installer
+$packageArgs['file'] = Get-ChocolateyWebFile @packageArgs
+
+Start-Sleep 2
+
+$sigValid = (Get-AuthenticodeSignature -FilePath $packageArgs['file']).Status -eq 'Valid'
+
+try {
+    if ($sigValid) {
+        $toolName = 'Google Chrome'
+        $category = 'Utilities'
```

Comment:
Are you using the category? I think we don't need to add Chrome to any category, just install it in the system, as it is a dependency for other tools (cyberchef) not really an analysis tool.

Reply:
it is still here 👀